### PR TITLE
Updated Chen-2019-171

### PR DESCRIPTION
**Pull request by mathildavz**
I have mapped some unlinked concepts in Chen-2019-171, according to the google sheet.

### CODE REVIEW

**Comment by LinguList** [comment]
The cases you mention are convincing, I'd suggest to UNMAP all dubious cases. This is no real loss, rather a win in clarity. Thanks!

**Comment by mathildavz** [comment]
Done :)

**Comment by LinguList** [comment]
The word is mainly from classical Chinese, as far as I know, referring 
exactly to something covering green and blue and turquoise.

File: concepticondata/conceptlists/Chen-2019-171.tsv [hunk 1]
```diff
@@ -96,17 +96,17 @@ Chen-2019-171-94	94	齊	齐	qi2	neat	0.333333333	4.611111111	2.555555556	0	1.8	V
 Chen-2019-171-95	95	乾	乾	gan1	dry	2.161290323	2.559139785	0.303030303	0.383838384	4.424242424	Haptic	0.419182713	1398	DRY
 Chen-2019-171-96	96	淺	浅	qian3	shallow	1.208333333	3.461538462	1	1.428571429	1.941176471	Visual	0.272305533	193	SHALLOW
 Chen-2019-171-97	97	強	强	qiang2	strong	2.416666667	3.461538462	5	2.125	1.846153846	Auditory	0.212389381		
-Chen-2019-171-98	98	青	青	qing1	green	0.538461538	5	0	0.615384615	0.277777778	Visual	0.777408638		
+Chen-2019-171-98	98	青	青	qing1	green	0.538461538	5	0	0.615384615	0.277777778	Visual	0.777408638	2382	BLUE OR GREEN
```
[the remark this answers is not in the document]
Comment (AnnikaTjuka) [reply concepticondata/conceptlists/Chen-2019-171.tsv:99]: Okay, then you can ignore my comment here, @mathildavz.

**Comment by mathildavz** [comment]
I unmapped Chen-2019-171-73. Are there any other changes to be made?

**Review by AnnikaTjuka — APPROVED** [review]
Looks good! This can be merged.

**Pull request merged**